packages/babel-core/test/errors-stacks.js

```diff
@@ -283,4 +297,29 @@ describe("@babel/core errors", function () {
           at ... internal jest frames ..."
     `);
   });
+
+  (semver.gte(process.version, "12.0.0") && !USE_ESM ? it : it.skip)(
+    "should not throw in `node --frozen-intrinsics`",
+    function () {
+      expect(
+        spawnSync(
+          process.execPath,
+          [
+            "--frozen-intrinsics",
+            "-e",
+            `
+        const babel = require("../lib/index.js");
+        babel.parseSync("foo;", {
+          root: String.raw\`${fixture("valid")}\`,
+        });
+        console.log("%done%");
+        `,
+          ],
+          {
+            cwd: __dirname,
+          },
+        ).output + "",
```

Comment:
```suggestion
        spawnSync(
          process.execPath,
          [
            "--frozen-intrinsics",
            "--input-type=module",
            "-e",
            `
        import * as babel from "@babel/core";
        await babel.parseAsync("foo;", {
          root: String.raw\`${fixture("valid")}\`,
        });
        console.log("%done%");
        `,
          ],
          { cwd: __dirname },
        ).output + "",
```

Does this work with ESM?